src/services/user-config-service.test.ts

```diff
@@ -24,7 +24,13 @@ describe("User Config Service", () => {
 		"      - \"Produktion\"\n" +
 		"      - \"produção\"\n" +
 		"      - \"продакшн\"\n" +
-		"      - \"PROD-*\"";
+		"      - \"PROD-*\"\n" +
+		"  services:\n" +
```

Comment:
So, that's content. Where do we have assertions that these services are actually available via the service?